src/components/TextField/TextField.tsx

```diff
@@ -92,6 +103,9 @@ export const useStyles = makeStyles(
         color: theme.palette.black[400],
       },
     },
+    inputHasAdornment: {
+      maxWidth: theme.pxToRem(152),
+    },
```

Comment:
I'm curious about the reasoning for this class - below it looks like we are using padding to shift the cursor for the input left / right depending on the adornment positioning (start/end). Are the `padding` attributes not enough in this case where it'll extend the width of the element?

Reply:
Correct, without this `maxWidth`, `<`TextField`s with adornments and without `fullWidth` will have unequal widths as compared to `<TextFields`s without adornments.

Without `maxWidth` widths are uneven
<img width="307" alt="Screen Shot 2021-08-10 at 3 23 44 PM" src="https://user-images.githubusercontent.com/485903/128922412-317e651f-bf76-41c5-8e44-971a19417205.png">

Reply:
Ah nice thanks for the info!